tutorials/gramex.yaml

```diff
@@ -16,3 +16,60 @@ url:
     kwargs:
       url: $YAMLPATH/store-sales.csv
       modify: data.groupby(['Category', 'Region'])['Sales'].sum().reset_index()
+
+  quickstart-output-steps:
+    pattern: /$YAMLURL/quickstart/output/([0-9]+)/index.html
+    handler: FileHandler
+    kwargs:
+      path: $YAMLPATH/quickstart/output/index.html
+      transform:
+        "index.html":
+          function: template
+          args: =content
+          kwargs:
+            handler: =handler
+      Content-Type: text/html
+      Cache-Control: no-store
+
+  quickstart-output-steps-source:
+    pattern: /$YAMLURL/quickstart/output/([0-9]+)/index.html.source
+    handler: FileHandler
+    kwargs:
+      path: $YAMLPATH/quickstart/output/index.html
+      transform:
+        "index.html":
+          function: template
+          args: =content
+          kwargs:
+            handler: =handler
+      headers:
+        Content-Type: text/plain
+        Cache-Control: no-store
+
+  g1-dropdown-output-steps:
+    pattern: /$YAMLURL/g1dropdown/output/([0-9]+)/index.html
+    handler: FileHandler
+    kwargs:
+      path: $YAMLPATH/g1-dropdown/output/index.html
+      transform:
+        "index.html":
+          function: template
+          args: =content
+          kwargs:
+            handler: =handler
+      Content-Type: text/html
+      Cache-Control: no-store
+
+  dashboard-output-steps:
+    pattern: /$YAMLURL/dashboards/output/([0-9]+)/index.html
+    handler: FileHandler
+    kwargs:
+      path: $YAMLPATH/dashboards/output/index.html
+      transform:
+        "index.html":
+          function: template
+          args: =content
+          kwargs:
+            handler: =handler
+      Content-Type: text/html
+      Cache-Control: no-store
```

Comment:
@jaidevd -- `index.html.source` endpoints are missing are dropdown and dashboards. When you add them, you could consider making three of them generic in one endpoint? `/$YAMLURL/{dashboards|dropdown|other}/output/([0-9]+)/index.html`